src/OrbitGl/include/OrbitGl/CallTreeView.h

```diff
@@ -188,20 +188,42 @@ class CallTreeUnwindErrorType : public CallTreeNode {
   orbit_client_data::CallstackType error_type_;
 };
 
-class CallTreeView : public CallTreeNode {
+class CallTreeView {
  public:
   [[nodiscard]] static std::unique_ptr<CallTreeView> CreateTopDownViewFromPostProcessedSamplingData(
       const orbit_client_data::PostProcessedSamplingData& post_processed_sampling_data,
-      const orbit_client_data::ModuleManager& module_manager,
-      const orbit_client_data::CaptureData& capture_data);
+      const orbit_client_data::ModuleManager* module_manager,
+      const orbit_client_data::CaptureData* capture_data);
 
   [[nodiscard]] static std::unique_ptr<CallTreeView>
   CreateBottomUpViewFromPostProcessedSamplingData(
       const orbit_client_data::PostProcessedSamplingData& post_processed_sampling_data,
-      const orbit_client_data::ModuleManager& module_manager,
-      const orbit_client_data::CaptureData& capture_data);
+      const orbit_client_data::ModuleManager* module_manager,
+      const orbit_client_data::CaptureData* capture_data);
+
+  CallTreeView(const orbit_client_data::ModuleManager* module_manager,
+               const orbit_client_data::CaptureData* capture_data)
+      : module_manager_{module_manager}, capture_data_{capture_data} {}
 
-  CallTreeView() : CallTreeNode{nullptr} {}
+  [[nodiscard]] CallTreeNode* GetCallTreeRootNode() { return &call_tree_root_; }
+  [[nodiscard]] const CallTreeNode* GetCallTreeRootNode() const { return &call_tree_root_; }
```

Comment:
Kind of irrelevant: The name you game means "get the root node of the tree". However, since this returns a `CallTreeNode`, I would maybe expect this to be called `GetRootCallTreeNode`, i.e., "get the root `CallTreeNode`".

Reply:
Done.